1.8/administration/installing/custom/graceful-shutdown.md

```diff
@@ -0,0 +1,12 @@
+---
+post_title: Graceful Node Shutdown
+menu_order: 501
+---
+
+You can use these instructions to remove a DC/OS cluster node.
+
+1.  Add a [Mesos maintenance primitive](http://mesos.apache.org/documentation/latest/maintenance/) for the cluster node to be removed (via /machine/down). The machine will not be used for new tasks.
+1.  Use the Mesos KillPolicy ([MESOS-4909](https://issues.apache.org/jira/browse/MESOS-4909)) to specify how long, in seconds, [Marathon](https://github.com/mesosphere/marathon/commit/4a6cd7afe66c91741d835a55933da67b72c331de) will wait after sending a SIGTERM before killing the process via SIGKILL.
```

Comment:
Instead of the JIRA, try: https://github.com/apache/mesos/blob/1.1.x/include/mesos/mesos.proto#L434-L461